### PR TITLE
Add/member support

### PR DESCRIPTION
**Pull request by khemrajrathore**
- Adding Member tagging for python, Java
- fix - debug mode in logging not working
- Added importsPass in Python

### CODE REVIEW

**Review by pandurangpatil — CHANGES_REQUESTED** [review]
@khemrajrathore You have made changes related to default semantics usage. The changes look good to me.  However, let's run this branch through a comparison script in order to make sure there is no diff.

cc: @dbMundada

File: build.sbt [hunk 1]
```diff
@@ -44,7 +43,10 @@ libraryDependencies ++= Seq(
   "com.networknt"                    % "json-schema-validator"   % "1.0.72",
   "com.fasterxml.jackson.module"    %% "jackson-module-scala"    % jacksonVersion,
   "com.fasterxml.jackson.dataformat" % "jackson-dataformat-yaml" % jacksonVersion,
-  "com.github.wnameless.json"        % "json-flattener"          % "0.14.0"
+  "com.github.wnameless.json"        % "json-flattener"          % "0.14.0",
+  "org.apache.logging.log4j"         % "log4j-core"              % "2.19.0",
+  "org.apache.logging.log4j"         % "log4j-api"               % "2.19.0",
```
Comment (pandurangpatil) [inline build.sbt:48]: @khemrajrathore remove this dependency of `log4j-api`; this one is not required.

**Review by pandurangpatil — CHANGES_REQUESTED** [review]
Make requested change.

**Comment by khemrajrathore** [comment]
Have removed `log4j-api` dependency

**Review by pandurangpatil — APPROVED** [review]

**Pull request merged**